src/classmentors/components/cohorts/cohorts.js

```diff
@@ -8,7 +8,10 @@ import cohortViewTmpl from './cohorts-view-cohort.html!text';
 import cohortEditTmpl from './cohorts-edit-cohort.html!text';
 import cohortStatsPageTmpl from './cohorts-view-cohort-stats-page.html!text';
 import cohortRankingPageTmpl from './cohorts-view-cohort-ranking-page.html!text';
+import c3 from 'c3';
+import d3 from 'd3';
 import './cohorts.css!';
+import '../../../jspm_packages/npm/c3@0.4.11/c3.css!';
```

Comment:
It should be `import 'c3/c3.css!;` or better it should be defined as a dependency of c3 and be implicitly loaded when you import c3.